src/index.html

```diff
@@ -6,9 +6,54 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+          src="images/logo.png"
+          alt="Moyo header"
+          class="logo__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="nav__link" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="nav__link" class="nav__link">Samsung</a>
+          </li>
```

Comment:
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a href="nav__link" class="nav__link is-active">Apple</a>
          </li>
          
          <li class="nav__item">
            <a href="nav__link" class="nav__link">Samsung</a>
          </li>
```
add blank lines between blocks of code